static/css/_variables.scss

```diff
@@ -1,16 +1,22 @@
 // Color Scheme
-$background: #F4F4F4;
-$form-input-background: #FFFFFF;
-$gray: #E3E3E3;
-$lightgray: #ECECEC;
+$gray: #F3F3F3;
 $black: #242424;
-$white: #F4F4F4;
+$white: #FDFDFD;
```

Comment:
this value might be a bit too close to actual white : pages that are mainly forms (creating article, editing profile, or even logging in) look odd in my opinion. I've tried with `F8F8F8`, and I think it looks better so to contrast with `FFFFFF` inside input elements